build.rs

```diff
@@ -1,5 +1,25 @@
 extern crate pkg_config;
 
 fn main() {
-    pkg_config::find_library("alsa").unwrap();
+    let missing: String = String::from("\"pkg-config\" \"--libs\" \"--cflags\" \"alsa\"");
+
+    match pkg_config::probe_library("alsa") {
```

Comment:
You can use `if let Err(e) = pkg_config::probe_library("alsa")` to avoid some rightward drift here.